config/jobs/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -538,7 +538,7 @@ presubmits:
     decorate: true
     branches:
     - master
-    - release-1.2
+    - release-1.3
```

Comment:
Incremented all the release branch numbers. 
So our PRs will be tested after merging the branch into master and also into release-1.3